### ISSUE TITLE
[5.0.1] Android compile version error

### ISSUE THREAD

**Issue by SolMendiola** [bugsee/flutter-bugsee#15]
Android compile version is a String and it should be an int.

SDK version: 5.0.1

Log:
```text
* What went wrong:
A problem occurred evaluating project ':bugsee_flutter'.
> Unsupported value: 31. Format must be one of:
  - android-31
  - android-31-ext2
  - android-T
  - vendorName:addonName:31
```
It's defined as: 
```
android {
    compileSdkVersion bgsGetExtPropSafe('compileSdkVersion', '31')
```
It should be defined as

```
android {
    compileSdkVersion bgsGetExtPropSafe('compileSdkVersion', '33')
```

**Comment by mirland** [comment]
We copy-paste wrong the code 🙂 
The change should be 

```
android {
    compileSdkVersion bgsGetExtPropSafe('compileSdkVersion', 31)
```

**Comment by krassx** [comment]
Hi @mirland and @SolMendiola. Thank you for bringing this to our attention. We have fixed this in the recently published version 5.0.2.

Please try it and let us know how it goes
